packages/protobuf-test/src/message.test.ts

```diff
@@ -12,23 +12,57 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { PlainMessage, Timestamp } from "@bufbuild/protobuf";
+import { PartialMessage, PlainMessage, Timestamp } from "@bufbuild/protobuf";
+import { Example as TS_Example } from "./gen/ts/extra/example_pb.js";
+import { Example as JS_Example } from "./gen/js/extra/example_pb.js";
+import { describeMT } from "./helpers.js";
 
 describe("PlainMessage", () => {
   const plainTimestamp: PlainMessage<Timestamp> = Timestamp.now();
   test("keeps regular fields", () => {
     // Regular fields are untouched.
     expect(plainTimestamp.nanos).toBeDefined();
   });
-  test("removes standard methods from type system", () => {
+  test("removes standard methods and wkt methods from type system", () => {
     // We want to test that the type system sees this function as undefined even though it's still actually there.  So
     // we expect TS error  TS2339, but add a simple test so Jest doesn't complain there's no expectations.
     // @ts-expect-error TS2339
     expect(plainTimestamp.toBinary).toBeDefined();
-  });
-  test("removes wkt methods from type system", () => {
     // Custom methods of well-known types are removed as well.
     // @ts-expect-error TS2339
     expect(plainTimestamp.toDate).toBeDefined();
   });
 });
+
+describe("PartialMessage", () => {
```

Comment:
constructor.test.ts already tests most of this, because constructors take a partial message as input.

Perhaps it makes sense to move the additional tests for PartialMessage there? The message being tested (protobuf_test_messages.proto3.TestAllTypesProto3) also has fields for all WKT.

Reply:
I switched to use `TestAllTypesProto3`, but if it's cool with you, I think I'd like to leave the tests here.  The `constructor.test` stuff is so constructor-specific and I like having this separate file to focus on the removal of standard/wkt methods and the recursiveness of that functionality.